utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterConstants.kt

```diff
@@ -16,3 +16,10 @@ annotation class SplashScreenWelcomeMsg
 val SPLASH_SCREEN_WELCOME_MSG = "splash_screen_welcome_msg"
 val SPLASH_SCREEN_WELCOME_MSG_DEFAULT_VALUE = false
 val SPLASH_SCREEN_WELCOME_MSG_VALUE = true
+
+@Qualifier
+annotation class MyDownloads
+
+val MY_DOWNLOADS_FLAG = "my_downloads_flag"
+val MY_DOWNLOADS_IS_ENABLED_DEFAULT = true
+val MY_DOWNLOADS_IS_DISABLE = false
```

Comment:
This shouldn't be needed, I think. The default is sufficient since it defines the compile-time on/off status for the flag.

Reply:
fixed